ddl/index.go

```diff
@@ -1850,6 +1850,10 @@ func (w *worker) addTableIndex(t table.Table, reorgInfo *reorgInfo) error {
 			if err != nil {
 				break
 			}
+			w.ddlCtx.mu.RLock()
+			w.ddlCtx.mu.hook.OnUpdateReorgInfo(reorgInfo.Job, reorgInfo.PhysicalTableID)
+			w.ddlCtx.mu.RUnlock()
```

Comment:
```suggestion
			defer w.ddlCtx.mu.RUnlock()
```
to ensure release lock.

Reply:
No, inside the loop we cannot use "defer". Otherwise it will block `RLock` forever.